frontends/concrete-python/concrete/fhe/compilation/circuit.py

```diff
@@ -328,3 +328,14 @@ def global_p_error(self) -> int:
             self.enable_fhe_execution()
 
         return self.server.global_p_error
+
+    @property
+    def total_tlu_count(self) -> int:
+        """
+        Get the total number of table lookups in the circuit.
+        """
+
+        if not hasattr(self, "server"):  # pragma: no cover
+            self.enable_fhe_execution()
+
+        return self.server.total_tlu_count
```

Comment:
This can actually be fetched from simulation as well, and it's also related to other properies in Circuit as well, so a couple of them can be accessed in simulation if available maybe?

Reply:
Done.